catalog/schematron/catalog-schematron-plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.0.0">
+
+    <cm:managed-service-factory
+            id="ddf.services.schematron.SchematronValidationService"
+            factory-pid="ddf.services.schematron.SchematronValidationService"
+            interface="ddf.catalog.validation.MetacardValidator">
+        <cm:managed-component class="ddf.services.schematron.SchematronValidationService"
+                              init-method="init" destroy-method="destroy">
+            <cm:managed-properties persistent-id="" update-strategy="container-managed"/>
```

Comment:
fwiw, I've noticed the same thing. I had the same empty `cm:managed-properties` and it didn't work without it